src/features/dashboard/Dashboard.tsx

```diff
@@ -1,13 +1,45 @@
-import { Typography } from '@mui/material';
+import {Box, Button, Card, CardActions, CardContent, Theme, Typography} from '@mui/material';
 import React from 'react';
-import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { RouteComponentProps, withRouter, Link as RouterLink } from 'react-router-dom';
+import {createStyles, makeStyles} from "@mui/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        card: {
+            border: 'solid 2px',
+            marginRight: theme.spacing(2),
+            borderColor: theme.palette.primary.main
+        }
+    }));
 
 const Dashboard: React.FunctionComponent<RouteComponentProps<any>> = () => {
+    const classes = useStyles();
 
     return (
-        <div>
-            <Typography>Dashboard</Typography>
-        </div>
+        <Box display="flex" position="relative" width={1} height={1}>
+            <Card className={classes.card} sx={{ maxWidth: 345 }}>
+                <CardContent>
+                    <Typography gutterBottom variant="h5" component="div">
+                        5 Skjema
+                    </Typography>
+                </CardContent>
+                <CardActions>
+                    <Button size="small" variant="outlined" component={RouterLink} to="/overview">Se integrasjoner</Button>
+                    <Button size="small" variant="outlined" component={RouterLink} to="/integration_configuration">Ny integrasjon</Button>
```

Comment:
Separate concepts with `/`.  This should probably be `/integration/configuration` or event (better?) `/integration/configuration/new`

Reply:
agree